src/client/components/allBlocks/allBlocks.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import connect from 'react-redux/es/connect/connect';
+import {
+  setVisibility,
+} from '../../actions/blocksActions';
+import BlockOfSliders from '../blockOfSliders/index';
+import Button from '../button/index';
```

Comment:
Why you call index file, if webpack get this file by default)

Reply:
fixed